tests/unit/test_clidriver.py

```diff
@@ -383,100 +414,6 @@ def inject_command_schema(self, command_table, session, **kwargs):
 
         command_table['foo'] = command
 
-    def test_aws_with_endpoint_url(self):
```

Comment:
Do we have anything to replace these tests? If not, we could just make these in terms of clients. Also some of the tests that were removed involved clients.